apps/condo/domains/news/schema/NewsItemRecipientsExportTask.test.js

```diff
@@ -113,12 +113,15 @@ describe('NewsItemRecipientsExportTask', () => {
             test('admin can', async () => {
                 const [objCreated] = await createTestNewsItemRecipientsExportTask(adminClient, adminClient.user, dummyO10n)
 
-                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id)
-
+                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id, {
```

Comment:
If the worker manages to process the task, this string will fail with an exception.
<img width="861" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/80a1fe05-cf1f-46e7-909d-1fcedbaa7a1b">